### ISSUE TITLE
ui-collapsible not opening the content initially if it is inside ui-modal

### ISSUE THREAD

**Issue by rohanil** [JosephusPaye/Keen-UI#354]
When I use `<ui-collapsible open>`, it is expected to be opened initially. But in my case, the arrow is upwards (just like an opened collapsible) but the content is not opened. Is there any bug in `ui-collapsible` component?
I am using ui-collapsible inside ui-modal. And once modal is opened, ui-collapsible is expected to show content initially.

**Comment by JosephusPaye** [comment]
Hi @rohanil,
 
The way UiCollapsible opens/closes has been refactored in https://github.com/JosephusPaye/Keen-UI/commit/435677b1bb8f3fef5c78cf048d05f42b6286823a#diff-02e71135a52f3e967db20aca8a8000c9. 

That should fix this problem. It's on `next` and will be released soon.

To try it:

```sh
# npm
npm install keen-ui@JosephusPaye/Keen-UI#next

# yarn
yarn add keen-ui@JosephusPaye/Keen-UI#next
```

See the [changelog](https://github.com/JosephusPaye/Keen-UI/blob/next/CHANGELOG.md) for all changes.

**Comment by JosephusPaye** [comment]
Fix released in [v1.1.0](https://github.com/JosephusPaye/Keen-UI/releases/tag/v1.1.0).
